server/app/services/question/types/QuestionType.java

```diff
@@ -21,21 +21,21 @@
 
 /** Defines types of questions supported. */
 public enum QuestionType {
-  ADDRESS("Address Field", AddressQuestion.class),
+  ADDRESS("Address", AddressQuestion.class),
```

Comment:
praise: Great idea to make these labels more concise and to-the-point.  I think it makes them more user-friendly.